### DISCUSSION TITLE
Having problem with [post-text]

### DISCUSSION THREAD

**Discussion by DiegoChito** [kevinzg/facebook-scraper#1014] [Q&A] [answered]
Recently I was scraping some facebook profiles. It was looking very good, but when I saw the column for "post-text" for the first post was getting concated with the others post and a lot of info that honestly doesn's understood. 

I'm posting an example:

"(function(){var n=now_inl();requireLazy(["__bigPipe"],function(bigPipe){bigPipe.beforePageletArrive("tlFeed",n);})})(); requireLazy(["__bigPipe"],(function(bigPipe){bigPipe.onPageletArrive({displayResources:["Q7npRpk","jMJUy0e","lWMrrKH","VlTh/BP","RzbQlJH","MmIV9rr","IN0FL4P","9mD2Jb8","PiOVcGT","hURtgcm","1zryJWw","FJqL2Al"],content:{tlFeed:{container_id:"u_0_2k_MK"}},id:"tlFeed",phase:4,last_in_phase:true,all_phases:[63,7,2,3,4],tti_phase:4,jsmods:{define:[["MUFIConfig",[],{hasIEMobileFixes:false,canReactToComment:true,hasUpdatedDock:false,photoComments:{dtsg:"NAcO-CTFoY5yn_gCFC0jCy-5tiF5_JGZT4NaZTnH1BfEOXSUfxWMeWw:34:1681851787",uploadDomain:"https://upload.facebook.com/",waterfallAppName:"web_m_touch",waterfallSource:"photo_comment"},vpvLoggingTimeout:300,actionSounds:{like:null}},669],["BackgroundSyncParameters",[],{shouldRecoverReactions:true,recoverReactionsQe:""},3506],["MqttWebDeviceID",[],{clientID:"f7d3060d-2b61-45c3-86f5-4aae87ba3053"},5003],["UFIReactionIcons",[],{"1":{"16":{sprited:1,spriteCssClass:"sx_be1fb3",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/16x16/like.png"},"14":{sprited:1,spriteCssClass:"sx_fdc389",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/14x14/like.png"}},"2":{"16":{sprited:1,spriteCssClass:"sx_1e87ed",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/16x16/love.png"},"14":{sprited:1,spriteCssClass:"sx_2b8f86",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/14x14/love.png"}},"16":{"16":{sprited:1,spriteCssClass:"sx_b73584",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/16x16/support.png"},"14":{sprited:1,spriteCssClass:"sx_4431f0",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/14x14/support.png"}},"4":{"16":{sprited:1,spriteCssClass:"sx_d85109",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/16x16/haha.png"},"14":{sprited:1,spriteCssClass:"sx_2734e3",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/14x14/haha.png"}},"3":{"16":{sprited:1,spriteCssClass:"sx_d98142",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/16x16/wow.png"},"14":{sprited:1,spriteCssClass:"sx_0a5f12",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/14x14/wow.png"}},"7":{"16":{sprited:1,spriteCssClass:"sx_39946a",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/16x16/sorry.png"},"14":{sprited:1,spriteCssClass:"sx_9637c3",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/14x14/sorry.png"}},"8":{"16":{sprited:1,spriteCssClass:"sx_6d8f5a",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/16x16/anger.png"},"14":{sprited:1,spriteCssClass:"sx_63e454",spriteMapCssClass:"sp_f9OLn5Y9NX-",origPath:"images/ufi/reactions/v7_DO_NOT_USE_DIRECTLY/14x14/anger.png"}}},1059],["UFIReactionTypes",[],{LIKE:1,ordering:[1,2,16,4,3,7,8],NONE:0,reactions:{"1":{color:"#2078f4",display_name:"Me gusta",is_deprecated:false,is_visible:true,name:"like",type:1},"2":{color:"#f33e58",display_name:"Me encanta",is_deprecated:false,is_visible:true,name:"love",type:2},"13":{color:"#f0ba15",display_name:"Selfie",is_deprecated:false,is_visible:false,name:"selfie",type:13},"11":{color:"#7e64c4",display_name:"¡Gracias!",is_deprecated:false,is_visible:true,name:"dorothy",type:11},"12":{color:"#ec7ebd",display_name:"Orgullo",is_deprecated:false,is_visible:true,name:"toto",type:12},"4":{color:"#f7b125",display_name:"Me divierte",is_deprecated:false,is_visible:true,name:"haha",type:4},"5":{color:"#f0ba15",display_name:"Me alegra",is_deprecated:true,is_visible:false,name:"yay",type:5},"3":{color:"#f7b125",display_name:"Me asombra",is_deprecated:false,is_visible:true,name:"wow",type:3},"10":{color:"#f0ba15",display_name:"Me confunde",is_deprecated:true,is_visible:false,name:"confused",type:10},"16":{color:"#f7b125",display_name:"Me"

Something like that show me in [post-text].

Also this alerts appers in m python:

![image](https://github.com/kevinzg/facebook-scraper/assets/47165694/5a246397-23b0-415e-85a4-d50fd2b4d39e)

**Comment by neon-ninja** [comment]
Try update lxml

**Reply by DiegoChito** [reply]
Im sorry but where? Can you show me?

**Reply by neon-ninja** [reply, marked as answer]
pip install -U lxml
